Vuologist/bumpie
# Forgot password page

Closes #124 
Closes #135 

Test instructions:
Make sure to already have an account created.

1. Go to /sign-in
2. Click I forgot my password
3. Enter an your email
4. You should see an icon and sentence that says a link was sent to your email.
5. Go to your mail. You should see a forgot password email for bumpie
6. Open the link given, copy /password-reset/email/code
7. Replace /sign-in with above URL
8. Enter your new password
9. If successful, you will be pushed to the sign in page
10. Login in. If successful, you should be pushed to dashboard
